ksqldb-streams/src/test/java/io/confluent/ksql/execution/streams/materialization/KsqlMaterializationTest.java

```diff
@@ -137,10 +94,69 @@
 
   @Before
   public void setUp() {
+    SCHEMA = LogicalSchema.builder()
+            .keyColumn(ColumnName.of("k0"), SqlTypes.STRING)
+            .valueColumn(ColumnName.of("v0"), SqlTypes.STRING)
+            .valueColumn(ColumnName.of("v1"), SqlTypes.STRING)
+            .build();
+
+    A_KEY = GenericKey.genericKey("k");
+    A_KEY2 = GenericKey.genericKey("k2");
+
+    A_ROWTIME = 12335L;
+
+    WINDOW_START_BOUNDS = Range.closed(
+            Instant.now(),
+            Instant.now().plusSeconds(10)
+    );
+
+    WINDOW_END_BOUNDS = Range.closed(
+            Instant.now().plusSeconds(1),
+            Instant.now().plusSeconds(11)
+    );
+    PARTITION = 0;
+
+    A_VALUE = GenericRow.genericRow("a", "b");
+    A_VALUE2 = GenericRow.genericRow("a2", "b2");
+    TRANSFORMED = GenericRow.genericRow("x", "y");
+    A_WINDOW = Window.of(Instant.now(), Instant.now().plusMillis(10));
+    STREAM_WINDOW = new TimeWindow(
+            A_WINDOW.start().toEpochMilli(),
+            A_WINDOW.end().toEpochMilli()
+    );
+
+    ROW = Row.of(
+            SCHEMA,
+            A_KEY,
+            A_VALUE,
+            A_ROWTIME
+    );
+
+    ROW2 = Row.of(
+            SCHEMA,
+            A_KEY2,
+            A_VALUE2,
+            A_ROWTIME
+    );
+
+    WINDOWED_ROW = WindowedRow.of(
+            SCHEMA,
+            new Windowed<>(A_KEY, STREAM_WINDOW),
+            A_VALUE,
+            A_ROWTIME
+    );
+
+    WINDOWED_ROW2 = WindowedRow.of(
+            SCHEMA,
+            new Windowed<>(A_KEY2, STREAM_WINDOW),
+            A_VALUE2,
+            A_ROWTIME
+    );
+
```

Comment:
These were pulled into the setup layer because `KsqlMaterialization.getIntermediateRow` mutates some of the variables